portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,102 @@
+html, body {
+  height: 100%;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
-#greeting-container {
+.page {
+  height: 100%;
+  margin: 0;
+  overflow: hidden;
+}
+
+h1 {
+  font-size: 500%;
+  text-align: center;
+}
+
+h2 {
+    font-size: 250%;
+}
+
+#home {
+  background-image: url('/images/toba-1-crop.jpg');
+  background-repeat: no-repeat;
+  background-size: 100%; 
+  padding-top: 3%;
+}
+
+#about {
+  background-image: url('/images/toba-memeoji2.jpg');
+  background-position: 80% 40%;
+  background-repeat: no-repeat;
+  background-size: 35%;
+}
+
+#about h2, p {
```

Comment:
In Google's style guide, we ask for multiple selectors to be split into their own lines. So:

#about h2,
p {
...
}